test/inductor/test_torchinductor.py

```diff
@@ -1275,7 +1275,9 @@ def fn(x):
             return torch.arange(0.1, 8.0001, 1, dtype=x.dtype, device=x.device)
 
         # Test that float arguments are truncated to int when dtype is set explicitly
-        make_arg = functools.partial(make_tensor, device="cpu", requires_grad=False)
+        make_arg = functools.partial(
+            make_tensor, device=self.device, requires_grad=False
```

Comment:
Every test here constructs the inputs on CPU and `self.common` will move it to the correct device. Might not be ideal but all tests do this.

Reply:
Oh, I didn't realize the inputs would be placed to CUDA by `self.common` (in `check_model_cuda`). If so, we don't need to change anything here then.

Reply:
Thank you for your comments! I will close this pull request.